### ISSUE TITLE
User-space drivers

### ISSUE THREAD

**Issue by delletenebre** [Arksine/Easycam#3]
Hello, may be it possible to adapt this (https://bentrask.com/easycap) "user-space" drivers to Android?

**Comment by anmaped** [comment]
Yes. It is possible. I'm interesting on producing a proper driver for utv007 to add to easycam.
